### ISSUE TITLE
SLF4J messages when using default zio-logging backend

### ISSUE THREAD

**Issue by carlosedp** [vitaliihonta/zio-temporal#25]
In my project I'm using the default zio-logging backend and when starting the app I get the following messages:

```
[47/47] hello.run
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
timestamp=2023-02-24T18:28:23.035899-03:00 level=INFO thread=zio-fiber-7 message="Started sample-worker"?
timestamp=2023-02-24T18:28:23.078098-03:00 level=INFO thread=zio-fiber-6 message="HTTP Server started at http://localhost:8082"
```

Since SLF4J is not used, can't these be supressed?

**Comment by carlosedp** [comment]
Found-out that just adding the `ch.qos.logback:logback-classic` library and a `logback.xml` config supresses these messages with no need to switch the logging framework from zio-logging to zio-logging-slf4j.

In my project, I added https://github.com/carlosedp/zio-temporal-hello/blob/5cf5d410307e194156e4f9d4ea78c64cc4107dca/build.sc#LL44C56-L44C56 to `build.sc` and https://github.com/carlosedp/zio-temporal-hello/blob/main/shared/resources/logback.xml config which is a bit different since I went with latest logback version.

Also added https://github.com/carlosedp/zio-temporal-hello/blob/5cf5d410307e194156e4f9d4ea78c64cc4107dca/build.sc#L26 to my `build.sc` to be able to use the same `resources` dir for all modules (worker, client and webclient) sharing the logback config.
